wgsl/analyze/Grammar.py

```diff
@@ -1801,6 +1834,22 @@ def __init__(self, json_text, start_symbol, ignore='_reserved'):
 
         # First decode it without any interpretation.
         pass0 = json.loads(json_text)
+
+        # Get the external tokens, these are not necessarily represented in the rules.
```

Comment:
Deal with the fact that the Treesitter JSON grammar now has tokens for the external scanner that are *not* represented in the main grammar.
(We never dealt with block comments before because they were not referenced by any other rules).